front/app/containers/LandingPage/HomepageInfoSection.tsx

```diff
@@ -37,17 +37,20 @@ const StyledQuillEditedContent = styled(QuillEditedContent)`
   }
 `;
 
-const HomepageInfoSection = () => {
-  const appConfiguration = useAppConfiguration();
+type Props = {
+  sectionData: 'bottom_info_section_multiloc' | 'top_info_section_multiloc';
+};
 
-  if (!isNilOrError(appConfiguration)) {
-    const homepageInfoMultiloc =
-      appConfiguration.data.attributes.homepage_info_multiloc;
+const HomepageInfoSection = ({ sectionData }: Props) => {
+  const homepageSettings = useHomepageSettings();
+  if (!isNilOrError(homepageSettings)) {
+    const homepageInfoMultiloc = homepageSettings.data.attributes[sectionData];
 
     if (homepageInfoMultiloc && !isEmptyMultiloc(homepageInfoMultiloc)) {
       return (
         <CustomSectionContentContainer>
           <StyledQuillEditedContent>
+            {/* does this need to be changed if there are two of them? */}
```

Comment:
I'm not familiar with our `Fragments` but I'm guessing these need to be unique strings? If so it might make sense to split this file into two with all of their data hardcoded, rather than what I've done here (one file with different values based on the key you pass in)